nomic/data_operations.py

```diff
@@ -24,6 +24,27 @@
 class AtlasMapDuplicates:
     """
     Atlas Duplicate Clusters State
+
```

Comment:
i know these were probably a pain to write up but this is very much appreciated and really improves on the docs 😄

Reply:
No, this is what the world needed